sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/physicianreport/AbstractPhysicianReportProcessingFlow.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2022 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package de.symeda.sormas.ui.externalmessage.physicianreport;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.caze.CaseDataDto;
+import de.symeda.sormas.api.caze.CaseSelectionDto;
+import de.symeda.sormas.api.externalmessage.ExternalMessageDto;
+import de.symeda.sormas.api.person.PersonDto;
+import de.symeda.sormas.api.person.PersonReferenceDto;
+import de.symeda.sormas.api.user.UserDto;
+import de.symeda.sormas.ui.externalmessage.processing.AbstractProcessingFlow;
+import de.symeda.sormas.ui.externalmessage.processing.PersonAndPickOrCreateEntryResult;
+import de.symeda.sormas.ui.externalmessage.processing.PickOrCreateEntryResult;
+import de.symeda.sormas.ui.externalmessage.processing.flow.ProcessingResult;
+import de.symeda.sormas.ui.externalmessage.processing.flow.ProcessingResultStatus;
+
+public abstract class AbstractPhysicianReportProcessingFlow extends AbstractProcessingFlow {
+
+	public AbstractPhysicianReportProcessingFlow(UserDto user) {
+		super(user);
+	}
+
+	public CompletionStage<ProcessingResult<CaseDataDto>> run(ExternalMessageDto externalMessage) {
+
+		//@formatter:off
+		return doInitialChecks(externalMessage)
+			.then((ignored) -> pickOrCreatePerson(externalMessage))
+			.then((p) -> pickOrCreateEntry(p.getData(), externalMessage))
+			.thenSwitch()
+				.when(PersonAndPickOrCreateEntryResult::isNewCase, (f, e) -> f
+						.then((ignored) -> createCase(e.getPerson(), externalMessage)).then((c) ->
+								convertSamePersonContactsAndEventparticipants(c.getData()).thenCompose((ignored) -> CompletableFuture.completedFuture(c)))
```

Comment:
```suggestion
								convertSamePersonContactsAndEventParticipants(c.getData()).thenCompose((ignored) -> CompletableFuture.completedFuture(c)))
```
Just changed the capitalization for `convertSamePersonContactsAndEventParticipants`